ballerina-command/src/main/java/org/ballerinalang/command/cmd/UpdateToolCommand.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ballerinalang.command.cmd;
+
+import org.ballerinalang.command.BallerinaCliCommands;
+import org.ballerinalang.command.util.ToolUtil;
+import picocli.CommandLine;
+
+import java.io.IOException;
+import java.io.PrintStream;
+import java.security.KeyManagementException;
+import java.security.NoSuchAlgorithmException;
+
+/**
+ * This class represents the "Update" command and it holds arguments and flags specified by the user.
```

Comment:
New class missing @ since tag.

Reply:
Seems all classes missed. Fixed for update command. We can others that in seperate pr.